lib/mjsonwp/mjsonwp.js

```diff
@@ -257,7 +257,9 @@ function buildHandler (app, method, path, spec, driver, isSessCmd) {
                `result: ${_.truncate(JSON.stringify(driverRes), LOG_OBJ_LENGTH)}`);
     } catch (err) {
       let actualErr = err;
-      if (!(isErrorType(err, MJSONWPError) ||
+      if (err instanceof errors.ProxyRequestUnsuccessfulError) {
```

Comment:
we actually can't use `instanceof` checks here because they will fail across package versions.

i.e., an instance of this error class for v1.2.3 or base-driver is not the same as one in v1.2.4. this is why we made the `isErrorType` function instead.